src/components/TransactionsChart.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/styles'
+import Typography from '@material-ui/core/Typography'
+import { ResponsiveContainer, YAxis, AreaChart, Area, Tooltip } from 'recharts'
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    backgroundColor: '#fff',
+    padding: theme.spacing(2),
+    borderRadius: theme.spacing(1),
+    boxShadow:
+      '0px 5px 5px -3px rgba(0,0,0,0.2), 0px 8px 10px 1px rgba(0,0,0,0.14), 0px 3px 14px 2px rgba(0,0,0,0.12)'
+  },
+  description: {
+    fontWeight: 'normal'
+  }
+}))
+
+export default ({ data }) => {
+  const classes = useStyles()
+
+  const renderCustomTooltip = ({ active, payload }) => {
+    if (active) {
+      return (
+        <div className={classes.wrapper}>
+          <Typography variant="h6">
+            Block:{' '}
+            <span className={classes.description}>
+              {' '}
+              {payload[0].payload.block}
+            </span>
+          </Typography>
+          <Typography variant="h6">
+            Transactions:{' '}
+            <span className={classes.description}>
+              {' '}
+              {payload[0].payload.transactions}
+            </span>
+          </Typography>
+        </div>
+      )
+    }
+
+    return null
+  }
+
+  return (
+    <>
```

Comment:
is necessary to wrap this return code into `<> <ResponsiveContainer ... </>` ??

Reply:
no it's not necessary, you are right at some point I had multiple elements in this return so I used the <> wrapper but then I removed it and I forget to remove the wrapper